src/Conversion/DocumentFormat.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Uploadcare\Conversion;
+
+use Uploadcare\Interfaces\Conversion\{ConversionFormatInterface};
+use Uploadcare\Interfaces\SerializableInterface;
+
+class DocumentFormat implements ConversionFormatInterface, SerializableInterface
+{
+    private array $conversionFormats = [];
+    private string $name = 'pdf';
+
+    public static function rules(): array
+    {
+        return [
+            'name' => 'string',
+            'conversionFormats' => 'array',
+        ];
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function setName(string $name): self
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getConversionFormats(): ?array
```

Comment:
Does this need to be nullable?  Seems like it's the only place that is.

```suggestion
    public function getConversionFormats(): array
```